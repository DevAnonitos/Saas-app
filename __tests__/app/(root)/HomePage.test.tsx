import Home from "@/app/(root)/page";
describe('Page Imports', () => {
    it('should import HomePage', () => {
        expect(Home).toBeDefined();
        expect(Home).toMatchSnapshot();
    });
});
