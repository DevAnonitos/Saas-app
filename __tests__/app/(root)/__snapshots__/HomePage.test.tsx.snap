// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Imports should import HomePage 1`] = `[Function]`;
